src/Kovan-DssSpell.t.sol

```diff
@@ -658,6 +658,10 @@ contract DssSpellTest is DSTest, DSMath {
         {
             uint normalizedBox = values.cat_box * RAD;
             assertEq(cat.box(), normalizedBox);
+            assertTrue(
+                (cat.box() >= RAD && cat.box() < BILLION * RAD) ||
```

Comment:
I think we shouldn't check for 0 case, as that means no liquidations at all (which I don't think it will be an intended action). And in case it is an intended action, they can tweak the test knowing they are doing it.
Regarding the range, we might want to be more conservative. I'd say we do something like:
`cat.box() >= MILLION * RAD && cat.box() < 50 * MILLION * RAD`.
@brianmcmichael @godsflaw what do you guys think?

Reply:
50 million might be a little low. It's probably fine to set it there for now knowing that we may have to update the test in the future though.

Reply:
But I agree, we should probably error if `box == 0`